### ISSUE TITLE
Question: Simplest way to hot reload a CSS file in Webpack using Webpack Dev Middleware

### ISSUE THREAD

**Issue by buzinas** [webpack/webpack#2540]
Hi,

I opened a question on Stack Overflow 11 days ago, but it didn't receive any comment nor answers, so I tried to open a bounty three days ago, but still no answers, so I decided to come here to see if that question can get some attention.

Please, if anyone has the time to take a look, I would really appreciate:
http://stackoverflow.com/questions/37203909/simplest-way-to-hot-reload-a-css-file-in-webpack-using-webpack-dev-middleware

Basically, I want to know if there is a simple way to hot reload a plain css file generated by gulp and incremental updated by it too. I don't want to migrate my Stylus build step to Webpack, and if possible, not even use css-loader or anything like that. Ideally, I would like to only hot reload the application any time the css file changes.

**Comment by bebraw** [comment]
Given there is an answer at SO, closing.

**Comment by buzinas** [comment]
The answer in SO doesn't solve the problem asked. That's why it isn't
marked as accepted.

Em 09/08/2016 02:25, "Juho Vepsäläinen" <notifications@github.com> escreveu:

**Comment by bebraw** [comment]
Even so, Stack Overflow is the right place for support. Hopefully someone in the know can comment there. 👍 

When it comes to hot loading, it's actually style-loader that implements the HMR interface in webpack. I don't see how it could work if you don't set up loaders against your Stylus files since then you don't get the interface by definition.

Instead of HMR, you could look into [browser-sync-webpack-plugin](https://www.npmjs.com/package/browser-sync-webpack-plugin) to see if it fits your use case better.
